main/test/ca/uwaterloo/flix/library/TestFile.flix

```diff
@@ -18,7 +18,23 @@ namespace TestFile {
 
     use File.{exists, isDirectory, isRegularFile, isReadable, isSymbolicLink,
         isWritable, isExecutable, readLines, readLinesWith, read, readWith,
-        readLinesIter, readLinesIterWith, readBytes, readBytesWith, readChunks};
+        readLinesIter, readLinesIterWith, readBytes, readBytesWith, readChunks,
+        stat};
+
+    /////////////////////////////////////////////////////////////////////////////
+    // stat                                                                    //
+    /////////////////////////////////////////////////////////////////////////////
+
+    @test
+    def testStat01(): Bool & Impure =
+        Result.isOk(stat("README.md"))
+
+    @test
+    def testStat02(): Bool & Impure =
+        match stat("README.md") {
+            case Ok(v) => v.fileType == File/FileType.File
+            case _ => false
+        }
```

Comment:
I can't really test the `modificationTime` and `accessTime` functions, since they will change, if the `README.md` file is changed. Is it okay to leave them untested to avoid creating conflicts?

Reply:
I'd recommend making a test where we call the function but only check that it's `Ok(something)`. That way we at least know that it doesn't crash/fail.